packages/spec-lib/package.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Should plugin tspd to generate the decorator signature like other package in this repo

Reply:
Same here. Can you please elaborate on the comment? Thanks

Reply:
https://github.com/microsoft/typespec/blob/d0666e643e56775d4fc7c53c814e37677530a80d/packages/xml/package.json#L33

Reply:
Done. Thanks